themes/aura/assets/product.js

```diff
@@ -245,11 +267,6 @@ function selectDefaultOptions(parentSection) {
         break;
     }
   });
-
-  const selectedVariant = getSelectedVariant(parentSection);
```

Comment:
FYI: I have removed this code since I notice isn't do anything just recall functions.

 Please can you confirm that by doing a hard QA and if I'm wrong correct me 🙏